src/database/standalone-sqlite.ts

```diff
@@ -2329,12 +2339,22 @@ export class StandaloneSqliteDatabaseWorker {
     dataOffset: number;
     dataSize: number;
   }) {
+    const currentTimestamp = currentUnixTimestamp();
+    const idBuffer = fromB64Url(id);
+    const parentIdBuffer = fromB64Url(parentId);
+
     this.stmts.data.insertNestedDataId.run({
-      id: fromB64Url(id),
-      parent_id: fromB64Url(parentId),
+      id: idBuffer,
+      parent_id: parentIdBuffer,
       data_offset: dataOffset,
       data_size: dataSize,
-      indexed_at: currentUnixTimestamp(),
+      indexed_at: currentTimestamp,
+    });
+
+    this.stmts.data.updateVerifiedDataId.run({
```

Comment:
@djwhitt do you think it make sense to run this update in saveNestedDataId or just get the `verified` value from parent while running `insertDataId`?

Reply:
Hmm... good question. I think it's probably better to do it in `insertDataId` and save a query.

Reply:
[`6ccf079` (#222)](https://github.com/ar-io/ar-io-node/pull/222/commits/6ccf079d351126718a8023c7939b17a68974207d)